Parse/PFObject.h

```diff
@@ -623,13 +623,13 @@ NS_REQUIRES_PROPERTY_DEFINITIONS
  @param objects The list of objects to fetch.
  @param error Pointer to an `NSError` that will be set if necessary.
  */
-+ (void)fetchAll:(PF_NULLABLE NSArray *)objects error:(NSError **)error;
++ (instancetype)fetchAll:(PF_NULLABLE NSArray *)objects error:(NSError **)error;
 
 /*!
  @abstract *Synchronously* fetches all of the `PFObject` objects with the current data from the server.
  @param objects The list of objects to fetch.
  */
-+ (void)fetchAllIfNeeded:(PF_NULLABLE NSArray *)objects;
++ (instancetype)fetchAllIfNeeded:(PF_NULLABLE NSArray *)objects;
```

Comment:
This one should have an `NSArray` return type, since the block has an `NSArray` as a result.